common/src/main/java/com/wynntils/models/territories/profile/TerritoryProfile.java

```diff
@@ -21,25 +21,22 @@
 import net.minecraft.ChatFormatting;
 import net.minecraft.core.Position;
 
-public class TerritoryProfile {
+public record TerritoryProfile(
```

Comment:
I don't think this class gains by being converted to a record. Being a record is more than just fulfilling the formal requirements. It also conveys "i am just a stupid data carrier". Basically, the ideal record has no additional methods. Having one, or maybe two (for large records) is okay. But this class has so much special knowledge, that we send the wrong signals by turning it into a record.

We could perhaps do that, but then we'd need to move the logic away into the TerritoryModel or something like that. It might be a good refactoring, or it might not. But just this step is bad.